tests/deprecated_api/test_remove_1-4.py

```diff
@@ -109,3 +109,30 @@ def test_v1_4_0_deprecated_metrics():
     with pytest.deprecated_call(match='will be removed in v1.4'):
         iou(torch.randint(0, 2, (10, 3, 3)),
             torch.randint(0, 2, (10, 3, 3)))
+
+    from pytorch_lightning.metrics.functional.classification import recall
+    with pytest.deprecated_call(match='will be removed in v1.4'):
+        recall(torch.randint(0, 2, (10, 3, 3)),
+               torch.randint(0, 2, (10, 3, 3)))
+
+    from pytorch_lightning.metrics.functional.classification import precision
+    with pytest.deprecated_call(match='will be removed in v1.4'):
+        precision(torch.randint(0, 2, (10, 3, 3)),
+                  torch.randint(0, 2, (10, 3, 3)))
+
+    from pytorch_lightning.metrics.functional.classification import precision_recall
+    with pytest.deprecated_call(match='will be removed in v1.4'):
+        precision_recall(torch.randint(0, 2, (10, 3, 3)),
+                         torch.randint(0, 2, (10, 3, 3)))
+
+    from pytorch_lightning.metrics.functional import precision
+    with pytest.deprecated_call(match='will be removed in v1.4'):
+        precision_recall(torch.randint(0, 2, (10, 3, 3)),
+                         torch.randint(0, 2, (10, 3, 3)),
+                         class_reduction='micro')
+
+    from pytorch_lightning.metrics.functional import recall
+    with pytest.deprecated_call(match='will be removed in v1.4'):
+        precision_recall(torch.randint(0, 2, (10, 3)),
+                         torch.randint(0, 2, (10, 3)),
+                         class_reduction='micro')
```

Comment:
these two seems to be duplicates?
```suggestion

```

Reply:
Ah I forgot to replace `precision_recall` with `precision` and `recall`. So that's why those lines weren't covered haha